test/provisioner/provision_ibmcloud.go

```diff
@@ -300,7 +332,28 @@ func (p *IBMCloudProvisioner) CreateCluster(ctx context.Context, cfg *envconf.Co
 		return err
 	}
 
-	// TODO, wait cluter ready, it takes long time
+	clusterReady := false
+	waitMinutes := 50
+	ibmcloudTrace(fmt.Sprintf("Waiting cluster %s available.\n", IBMCloudProps.ClusterName))
+	for i := 0; i <= waitMinutes; i++ {
+		ready, err := isClusterReady(IBMCloudProps.ClusterName)
+		if err != nil {
+			ibmcloudTrace(fmt.Sprintf("Err %s happened when retrieve cluster, try again...\n", err))
+			continue
+		}
+		if ready {
+			ibmcloudTrace(fmt.Sprintf("Cluster %s is available.\n", IBMCloudProps.ClusterName))
+			clusterReady = true
+			break
+		}
+		ibmcloudTrace(fmt.Sprintf("Slept %d seconds...\n", i*60))
```

Comment:
nit
```suggestion
		ibmcloudTrace(fmt.Sprintf("Waited %d seconds...\n", i*60))
```

Reply:
nit: Part of me wonders if showing the wait in minutes might be helpful instead of seconds as we are waiting in minute chunks and once you get to 1140 seconds my brain struggles to make sense of what that means?